### DISCUSSION TITLE
test react query

### DISCUSSION THREAD

**Discussion by JSDUNIYA** [orval-labs/orval#372] [Q&A] [answered]
How to test react query  using mock any examples please share?

**Comment by gavinthomas-valtech** [comment]
does this help?

https://www.js-howto.com/testing-react-query-with-mock-service-worker/

**Comment by gavinthomas-valtech** [comment, marked as answer]
and https://codesandbox.io/s/react-query-msw-test-forked-mircr9?file=/src/hooks/__tests__/useApi.test.js
